parlai/mturk/tasks/woz/backend/suggestions.py

```diff
@@ -126,6 +128,49 @@ def get_suggestions(
 
         return suggestions, possibly_wrong_item_selected
 
+    def get_suggestions(
+        self,
+        wizard_utterance: Text,
+        primary_kb_item: Dict[Text, Any],
+        secondary_kb_item: Dict[Text, Any] = None,
+        api_names: Optional[List[Text]] = None,
```

Comment:
Very nice! Just one thing: As the structure of the input changes here, could you change this one line where get_suggestions is called, and just pass the same api-name twice (as a placeholder, I'll implement it so that two different ones are passed). Otherwise this code will break when I run it.

Reply:
right, thats done. unfortunately i didn't realise i already had some other commits before pushing again...

the other changes are all for the suggestions models for the new scenarios and it should be harmless.